### PR TITLE
Updated sdk v0.6.0

### PR DESCRIPTION
**Pull request by lqgioan**
Updated readme file

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -97,6 +97,69 @@ class SampleViewController: UIViewController {
...
+                switch data.result {
+                case .success:
+                    result?(true)
+                case .failure:
+                    result?(false)
+                }
+            }
+        }
+        offer.presentPromotionOffersModally(from: self, offers: offerList, userProfile: user, pushClaimTokensTask: task) { [weak self] result in
+        self?.dismiss(animated: true) {
```
Comment (ngo275) [inline README.md]: Please fix the indent

File: README.md [hunk 2]
```diff
@@ -215,6 +278,44 @@ class SampleViewController: UIViewController {
 
 > **Important**: For the `pushClaimTokensTask` callback, you need to keep `credifyId` on your side. You have to send the `credifyId` to Credify SDK when you use the methods that require `credifyId`. E.g: call `bnpl.presentModally` method or create `CredifyUserModel` model.
 
+### The Service Instance
```
Comment (ngo275) [inline README.md]: Please avoid using "service instance". This is not a common word. We should say "Service detail"

File: README.md (hunk 1 above)
Comment (lqgioan) [reply README.md]: I fixed it

File: README.md (hunk 2 above)
Comment (lqgioan) [reply README.md]: I updated the text. I also update the Android MR too.

**Pull request merged**